packages/global-styles/src/components/global-styles-variations/index.tsx

```diff
@@ -104,7 +105,16 @@ const GlobalStylesVariations = ( {
 	onSelect,
 	splitPremiumVariations = true,
 	displayFreeLabel = true,
+	globalStylesInPersonalPlan,
 }: GlobalStylesVariationsProps ) => {
+	const premiumStylesDescription = globalStylesInPersonalPlan
+		? translate(
+				'Unlock premium styles and tons of other features with the Personal plan, or try them out now for free.'
```

Comment:
Ahh, we should've thought of this earlier, but I changed the 'premium styles' in those strings to 'custom styles' #78848. We will get a conflict when merging both, but more importantly, the string will be untranslated. I think you should just change it here and on line 115 and I'll throw away these particular strings in #78848

Reply:
Yes, we're also going to get other conflicts with some of my other PRs. I'm already counting on this PR being a (tiny) bit of a pain to merge.